aepsych/server/utils.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import logging
+import os
+import sys
+
+import aepsych.utils_logging as utils_logging
+import aepsych.database.db as db
+
+logger = utils_logging.getLogger(logging.INFO)
+
+def get_next_filename(folder, fname, ext):
+    n = sum(1 for f in os.listdir(folder) if os.path.isfile(os.path.join(folder, f)))
+    return f"{folder}/{fname}_{n+1}.{ext}"
+
+def parse_argument():
+    parser = argparse.ArgumentParser(description="AEPsych Utils!")
+
+    sub_parsers = parser.add_subparsers(dest="subparser")
+
+    database_parser = sub_parsers.add_parser("database")
+
+    database_parser.add_argument(
+        "-l",
+        "--list",
+        help="Lists available experiments in the database.",
+        action="store_true",
+    )
+    database_parser.add_argument(
+        "-d",
+        "--db",
+        type=str,
+        help="The database to use if not the default (./databases/default.db).",
+        default=None,
+    )
+
+    database_parser.add_argument(
+        "-u",
+        "--update",
+        action="store_true",
+        help="Update the database tables with the most recent columns and tables.",
+    )
+
+    args = parser.parse_args()
+    return args
+
+def run_utils(args):
+    logger.info("Starting AEPscyh Utils!")
+    try:
+        if args.subparser == "database":
```

Comment:
Let's get rid of the "database" argument entirely, and instead make a dedicated "aepsych_database" entry point.